release/models/optical-transport/openconfig-wavelength-router.yang

```diff
@@ -554,6 +583,20 @@ module openconfig-wavelength-router {
         is configured";
     }
 
+    leaf ase-recovery-hysteresis {
+      type decimal64 {
+        fraction-digits 2;
+      }
+      units dB;
+      description
+        "Defines the optical power offset value relative to the
+        ASE injection threshold at which point ASE should be removed
+        and the media-channel should be restored.
+        Also specifies the optical power offset relative to
+        injection threshold in dB above which a media channel will
+        be initially added to the system";
```

Comment:
Perhaps just an academic comment, but is this typically called hysteresis or a threshold in the optical industry?  I'm more used to seeing hysteresis used for 'debounce' values and/or when there is a time delay involved.